src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <div class="container">
```

Comment:
```suggestion
    <div class="container">
```
don't add empty spaces between parent and child elements